### PR TITLE
Styling change cards & NavBar

### LINKED ISSUE #139: [Homepage Dashboard]: Navbar, Layout Formatting for Diagnostic Container 3.22.20
Component 1: Upfront welcome

- [x] Logo to the left + text-logo to the right all on one 
- [x] Good morning, { firstName }
- [x] Today is, { dayOfWeek, month #, year 
- [x] Small black horizontal bar
- [x] Prompt that reads, either: 
         - “Let’s log your health for today” (if empty)
         - “Great job logging  your health today (if fulfilled)
         - Only shows if you have yet to log it, maybe say something like “Great job logging your health      for today!”

### PR DESCRIPTION
**Pull request by AdhamAH**
Changed the style of the cards 

fixes #139

Screenshot 2020-03-24 at 19 06 54

Added. some style to NavBar to be more interactive 

Screenshot 2020-03-24 at 19 07 04

### CODE REVIEW

File: client/src/components/NavBar.js [hunk 1]
```diff
@@ -24,32 +21,42 @@ const styles = {
...
-  // };
+  state = {
+    value: 'log',
+  };
 
-  // handleChange = (event, value) => {
-  //   console.log(value);
-  //   this.setState({ value });
-  // };
+  handleChange = (event, value) => {
```
Comment (friedger) [inline client/src/components/NavBar.js]: What is the motivation to take the navigation out of redux again?

Comment (friedger) [reply client/src/components/NavBar.js]: If by accident, this should be reverted. If on purpose this should go into a new PR

Comment (SomeMoosery) [reply client/src/components/NavBar.js]: Was this resolved? The Files Changed I'm seeing still has this PR removing redux

**Comment by SomeMoosery** [comment]
For the sake of time and getting things going before an alpha release for feedback Friday, we've decided to move forward without Redux in this component. [Going off the Redux docs](https://redux.js.org/advanced/usage-with-react-router), it seems like while Redux is the SSOT for data, Router is the SSOT for, well, routing - so it may not make sense to intermingle the two. 

Open to ideas and suggestions of course, happy to open a new issue if the need arises.

**Pull request merged**